api/hearings/data/mock.data.ts

```diff
@@ -106,11 +106,11 @@ const HEARING_DAY_SCHEDULE_11: HearingDayScheduleModel = {
 const CASE_HEARING_1: CaseHearingModel = {
```

Comment:
couldn't see the new field
"hearingRequestDateTime": <hearingRequestDateTime>,
is added into 
CaseHearingModel?

Reply:
What Jira ticket was this for?